include/envoy/network/io_handle.h

```diff
@@ -71,12 +72,14 @@ class IoHandle {
    * Send a message to the address.
    * @param slices points to the location of data to be sent.
    * @param num_slice indicates number of slices |slices| contains.
-   * @param address is the destination address.
+   * @param self_ip is the source address whose port should be ignored.
```

Comment:
What does it mean if it is null? Source is not set? Can you clarify?

Reply:
Commented that null means kernel should select the source address.